lib/elixir/lib/map.ex

```diff
@@ -10,7 +10,7 @@ defmodule Map do
 
   @type key :: any
   @type value :: any
-  @compile {:inline, fetch: 2, put: 3, delete: 2, has_key?: 2}
+  @compile {:inline, fetch: 2, put: 3, delete: 2, has_key?: 2, drop: 2, take: 2}
```

Comment:
Inlining only works inside the current module so this wouldn't do anything since drop and take are not used here.